src/modules/tests/test_data/test-data.actions.ts

```diff
@@ -2,9 +2,9 @@ import { Action } from '@ngrx/store';
 import { FaultPayload } from './test-data.models';
 import { Competencies } from './test-data.constants';
 
-export const ADD_DRIVING_FAULT = '[Competency] Add Driving Fault';
-export const ADD_SERIOUS_FAULT = '[Competency] Add Serious Fault';
-
+export const ADD_DRIVING_FAULT = '[TestData] Add Driving Fault';
```

Comment:
Should be Competency for this and ADD_SERIOUS_FAULT.

For Manoeuvres could we use Manoeuvres instead of Test Data